bot/utility/utility.py

```diff
@@ -95,6 +95,47 @@ async def about(self, ctx: commands.Context):
         embed.add_field(name="Links:", value=str_links)
         await ctx.send(embed=embed)
 
+    @commands.group(name='howto', invoke_without_command=True)
+    @command_log
+    async def howto(self, ctx: commands.Context, subcommand: Optional[str]):
+        """Handler for the `howto` command.
+
+        Explains how to do various things
+
+        Args:
+            ctx (discord.ext.commands.Context): The context in which the command was called.
+            subcommand (Optional[str]): The subcommand the user wants to invoke.
+        """
+        message = "Ich kenne diesen Befehl leider nicht. Hast du vielleicht einen der folgenden gemeint?\n" if subcommand else None
+
+        await ctx.send(message, embed=discord.Embed(
+            title="Howtos",
```

Comment:
Please change it to "Available HowTos" instead of only "Howtos". It then looks more "polished" in my opinion.